cmd/common.go

```diff
@@ -1,9 +1,3 @@
-/* Copyright (c) Fortanix, Inc.
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
```

Comment:
You should keep these license notices and make sure you add them to new files too